doc/manuals/work_queue/index.md

```diff
@@ -1222,6 +1222,56 @@ API](http://ccl.cse.nd.edu/software/manuals/api/html/work__queue_8h.html).
 
 ### Security
 
```

Comment:
At the beginning of the section, place the note that, by default, no authentication is done,
and so you can do SSL, password, or both.

Reply:
Also a bit that SSL provides manager-to-worker authentication, while password gives you worker-to-manager auth.

Reply:
Also, that reminds me that the warning message emitted by the manager should now mention using either password or ssl.